GDAPI/GDAPI/Utilities/Objects/GeometryDash/LevelObjects/SpecialObjects/Pads/Pad.cs

```diff
@@ -1,19 +0,0 @@
-﻿using GDAPI.Utilities.Attributes;
-using GDAPI.Utilities.Enumerations.GeometryDash;
-using GDAPI.Utilities.Information.GeometryDash;
-using GDAPI.Utilities.Objects.GeometryDash.LevelObjects.Interfaces;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace GDAPI.Utilities.Objects.GeometryDash.LevelObjects.SpecialObjects.Pads
-{
-    /// <summary>Represents a pad.</summary>
-    public abstract class Pad : OrbPad
-    {
-        /// <summary>Initializes a new instance of the <seealso cref="Pad"/> class.</summary>
-        public Pad() : base() { }
-    }
-}
```

Comment:
Why was this deleted?

Reply:
Git apparently preferred showing a deletion and an addition for this and an interface. Nothing was lost, otherwise the project wouldn't be built.